integreat_cms/cms/migrations/0033_event_datetime.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 3.2.14 on 2022-08-21 13:03
+
+import datetime
+from django.db import migrations, models
+
+# pylint: disable=unused-argument
+def start_and_end_init(apps, schema_editor):
+    """
+    Initalize the new event datetime fields 'start' and 'end' from the respective
+    existing fields *_date and *_time.
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    events = apps.get_model("cms", "Event")
+    for event in events.objects.all():
+        event.start = datetime.datetime.combine(event.start_date, event.start_time)
+        event.end = datetime.datetime.combine(event.end_date, event.end_time)
+
+
+class Migration(migrations.Migration):
+    """
+    Update the datetime fields from the event model
+    """
+
+    dependencies = [
+        ("cms", "0032_region_bounding_box"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="event",
+            options={
+                "default_permissions": ("change", "delete", "view"),
+                "default_related_name": "events",
+                "ordering": ["start"],
+                "permissions": (("publish_event", "Can publish events"),),
+                "verbose_name": "event",
+                "verbose_name_plural": "events",
+            },
+        ),
+        migrations.AddField(
+            model_name="event",
+            name="end",
+            field=models.DateTimeField(
+                default=datetime.datetime.now, verbose_name="end"
+            ),
+        ),
+        migrations.AddField(
+            model_name="event",
+            name="start",
+            field=models.DateTimeField(
+                default=datetime.datetime.now, verbose_name="start"
+            ),
+        ),
```

Comment:
Makes sense. I think this might effect the internal representation in the database and in what order it is handled. So I am not sure of the effect of it.